addons/l10n_nz/data/account_tax_report_data.xml

```diff
@@ -95,9 +95,33 @@
                         <field name="expression_ids">
                             <record id="tax_report_box13_formula" model="account.report.expression">
```

Comment:
On existing databases, since you're using the same xmlid, the former 'external' expression will now become an aggregation. Because of that, the manual values that would have been made for it so far will all be lost (they have a relational field to the expression they target, and it will not target the expression denoted by tax_report_box13_manual)

==> Just keep tax_report_box13_formula for the 'external' expression, and call this one something like tax_report_box13_aggregation